### ISSUE TITLE
Automatic generate version with GitHub Actions

### ISSUE THREAD

**Issue by gumaciel** [Poing-Studios/godot-admob-editor#59]
### Is your feature request related to a problem? If so, please describe.

Automatize process

### What feature or improvement would you like to see?

```mermaid
flowchart TD

A[Runs workflow manually]
B[Checkout on branches 3.5/4.0, etc...]
C[Compress content of branches]
D[Generate semver 'Semantic Version']
E[Create Artifactory of contents with version]
F[Create a new release with tag version generated]

A --> B --> C --> D --> E --> F
```

**Comment by gumaciel** [comment]
To-do: 
- Update workflow `pull_request` to add tag-prefix: `godot?-v?.?.?.-alpha?`
- Verify branch name of new workflow when generate version and to check if is "3" or "4" or whatever

**Comment by gumaciel** [comment]
Closed on 84e5086a7800e9d95b7ce870a6b3c2608c5b81b5
